as-vending/res/configuration.yaml

```diff
@@ -0,0 +1,13 @@
+# Copyright © 2023 Intel Corporation. All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+
+Writable:
+  LogLevel: INFO
+
+Service:
+  Host: localhost
+  Port: 48099
+  StartupMsg: This microservice checks if ID numbers from REST requests are authenticated
+
+Trigger:
+  Type: http
```

Comment:
Since this is an app service the trigger needs to be message bus, which is done in common config, so we don't want to override it here. Also need to set the clientID for MQTT.

```suggestion
MessageBus:
  Optional:
    ClientId: "app-mqtt-export"

# Using default Trigger config from common config
```